charts/flyte-core/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: flyte-core
-description: A Helm chart for Flyte
+name: flyte
```

Comment:
We are keeping the existing names, right?

Reply:
@EngHabu I brought that up already somewhere but I can't find the other comment.

helm can have weird issues when a chart with a `-` is used as a subchart. So my suggestion is to use this overhaul as an opportunity to rename `flyte-core` to just `flyte` so it can be used as a subchart. The other one is basically the sandbox chart right? So it's not going to be used as a subchart and we could name it `flyte-sandbox`. We could bump the major version of the chart for the change.

Reply:
I gave it another try with the current `flyte-core` chart aliased as `flyte` and that seems to work fine. So perhaps things have improved since I tried last time.

```yaml
apiVersion: v2
appVersion: "1.0"
description: A Helm chart for Kubernetes
name: some-chart
version: 1.0.0

dependencies:
  - name: flyte-core
    alias: flyte
    version: 0.18.1
    repository: https://flyteorg.github.io/flyte
```

Reply:
@sbrunk it's nice. I will try it